src/components/vm/overview/watchdog.jsx

```diff
@@ -110,6 +112,11 @@ export const WatchdogModal = ({ vm, isWatchdogAttached, idPrefix }) => {
     };
 
     const save = (coldplug) => {
+        if (!watchdogAction) {
+            setValidationMessage(_("You need to select the action"));
+            return;
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1256-20231012-104029-22248714-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)